uber/tests/models/test_job.py

```diff
@@ -46,7 +46,8 @@ def test_non_volunteer(self, session):
 
     def test_restricted(self, session):
         assert session.assign(session.staff_two.id, session.job_six.id)
-        session.staff_two.trusted = True
+        session.staff_two.trusted_depts = str(c.CONSOLE)
+        # TODO: update
         session.commit()
         assert not session.assign(session.staff_two.id, session.job_six.id)
 
```

Comment:
oh nice, Ok didn't notice that conftest.py was defining a bunch of fixtures, all makes sense now.

interesting side effect I hadn't noticed but one current behavior is: someone can be trusted in a dept but not assigned to it.  I think this makes sense like just because i.e. Nick would be trusted to work in any dept, we wouldn't want him to show up as a person who could be assigned there if he wasn't explicitly signed up for that department.